CHANGES_NEXT_RELEASE

```diff
@@ -38,6 +38,7 @@
 - Fix: use X-Real-IP header as preferred option for from= field in log traces (#2353)
 - Fix: use op= instead of function= in log traces (along with moving the file/line identification from msg= to op=) (#2353)
 - Fix: reorder field in log traces lo align with IoTP Operations requirements (#2353)
+- Fix: bug that provoked old compound values of metadatas to be sent in notifications
```

Comment:
This problem didn't happen in 1.2.0 (as the whole compound metadata functionality has been recently developed). Thus, this line is not needed.